### PR TITLE
New "Most asked questions" section in FAQ

### PR DESCRIPTION
**Pull request by brifemu**
Draft PR to create the new section in FAQ page. Still in progress.

### CODE REVIEW

**Comment by Ein-Tim** [comment]
Was this requested by the RKI/BMG? 

Because in general FAQ means: "Frequently Asked Questions". Creating a new section which is titled "Most asked questions" does not really makes sense, IHMO. Instead the most frequently asked questions should be moved to the top & the FAQ should be clarified!

**Comment by dsarkar** [comment]
@Ein-Tim This was a request: A summary of the most common questions. The name of the section is tentative, it was initially "Top 10 questions". Suggestions welcome. Enjoy Friday evening and the weekend!

**Comment by Ein-Tim** [comment]
@dsarkar Ah, okay. 

I will think about it again. Enjoy the weekend!

**Comment by MikeMcC399** [comment]
@dsarkar 
Perhaps it would be better to start this off as an issue that we could discuss next week?

We should understand where these questions have come from exactly and how to best integrate them into the existing FAQs without overloading the user. 

The FAQs are already very long and I often find it difficult to locate information that I know is there.

I think also that there should be some consideration given to removing unneeded information (see also https://github.com/corona-warn-app/cwa-website/issues/2032 "Which versions to describe in FAQs?").  Anything which is only relevant to CWA 1.x and earlier 2.x versions should be removed.

**Comment by MikeMcC399** [comment]
Since this PR is in draft status I will not comment on details at the moment and I will wait until it is marked ready for review. Here are some general suggestions:

## Suggestions

1. If there is going to be a list, then use the same titles as:

- https://digitaler-impfnachweis-app.de/en/faq "Current most frequently asked questions"
- https://digitaler-impfnachweis-app.de/faq "Die aktuell häufigsten Fragen"

2. Limit the number of such articles in the list (less than 10)

3. Only have one article on each subject. If there is already an article on the subject in the main body of the existing FAQs, then combine the text into one article and move it to the new list.

4. If the new proposed text covers a description of how the app works which is already described in a section at the top level of https://www.coronawarn.app, then combine the texts, choose the best place to display the information and only include it one time. The main page already includes sections:
    - "This is how the app works best"
    - "How does the app work?"

5. Don't link to other articles by displaying a literal URL e.g. https://www.coronawarn.app/de/faq/#check_in. Instead use text such as the title of the article or its section heading to construct a hyperlink. Make all hyperlinks relative, not absolute.

6. In addition consider enhancing the in-app documentation. The description of the app's functionality displayed in the "Overview" menu item is very basic and in the meantime it is quite incomplete, due to functionality which has been added over the past 18 months. Putting this information into an FAQ instead of into the in-app documentation is an inferior workaround.

---
Just one detail point though:

There are a lot of instances of
`<figure>image<figcaption aria-hidden='true'><em>Abb. 1.</em></figcaption></figure>` which don't make sense in this context.

File: src/data/faq.json [hunk 1]
```diff
@@ -1,2819 +1,2893 @@
...
-                            "If these criteria are <b>not</b> met, an encounter is <b>not counted</b> within the app. It is a 'non-risk encounter'. These encounters will still show up in the operating system's contact log, but do not count towards the overall risk score in the app.",
-                            "If the conditions are met, the contact times are weighted depending on the signal attenuation:",
-                            "<ul><li>Times with an attenuation <63dB are weighted at 80%.</li>",
-                            "<li>Times with an attenuation >= 63dB and <73dB are weighted at 100%.</li>",
-                            "<li>Times with an attenuation >= 73dB and <79dB are weighted at 10%.</li></ul>",
-                            "Based on the transmission risk level (III to VIII), a factor between 0.6 and 1.6 is determined. The <i>weighted</i> exposure time from the previous step is multiplied by this factor, which then results in the <i>normalized</i> exposure time. Finally, all normalized times of a day are added up. Ultimately, this means that several short low risk encou
```
Comment (MikeMcC399) [inline src/data/faq.json]: "risk map" =>

"risk card"

"map" is an incorrect translation of "Karte" in German

Comment (MikeMcC399) [inline src/data/faq.json]: "Covid-19" => "COVID-19"

Comment (MikeMcC399) [inline src/data/faq.json]: "trash" = "recycle bin"

use the term which is used in the app

File: src/data/faq.json [hunk 2]
```diff
@@ -60,7 +60,6 @@
                             "You can find test sites linked to the Corona warning app here: <a href='map.schnelltestportal.de' target='_blank'>map.schnelltestportal.de</a>. Another overview of testing sites can be found on this page of the <a href='https://www.bundesanzeiger.de/pub/publication/EaknuyebqGE4JSI7pwk/content/EaknuyebqGE4JSI7pwk/BAnz%20AT%2021.09.2021%20V1.pdf?inline' target='_blank'>Federal Ministry of Health</a>. However, the test sites listed there are not guaranteed to be connected to the Corona-Warn-App.",
                             "<b>How can I warn?</b>",
                             "The idea behind the app is to warn other users of the app by voluntarily sharing your positive test result in the app. This will alert all users of the app that you have encountered in the past 14 days and have been in contact with long enough. For detailed information on the criteria for risk encounters, please refer to our FAQ: <a href='#encounter_criteria' target='_blank'>By which criteria are encounters evaluated and how do the recommendations for action differ?</a>",
-                            "//Reduction to 10 days in planning",
```
Comment (Ein-Tim) [inline src/data/faq.json]: That's nice to hear! 😁

File: src/data/faq.json (hunk 1 above)
Comment (MikeMcC399) [inline src/data/faq.json]: "convalescent" => "recovery"

"convalescent certificate" is not an official term, nor is this term used anywhere in the app

The term "Certificate of recovery" is legally defined by the [Regulation (EU) 2021/953 of the European Parliament and of the Council of 14 June 2021 on a framework for the issuance, verification and acceptance of interoperable COVID-19 vaccination, test and recovery certificates (EU Digital COVID Certificate) to facilitate free movement during the COVID-19 pandemic (Text with EEA relevance)](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32021R0953) in Article 7. In lists however, the document also alternatively refers for instance in Article 1 - Subject matter to "recovery certificates" as in the text "COVID-19 vaccination, test and recovery certificates (EU Digital COVID Certificate)" similar to the use on the title page of the Regulation itself.

Comment (MikeMcC399) [inline src/data/faq.json]: "You can find more information here:"

is missing a cross-reference

File: src/data/faq_de.json [hunk 3]
```diff
@@ -1,2833 +1,2907 @@
...
-                            "Seit Version 1.7 der Corona-Warn-App wird der Risikostatus bis zu 6 mal täglich aktualisiert, wenn eine WLAN-Verbindung vorhanden ist. Die wiederholte Abfrage wird automatisch durchgeführt. Bei einer mobilen Internetverbindung wird der Risikostatus aufgrund der <a href='#mobile_data_costs' target='_blank' >Zero-Rating-Verpflichtungen</a> der Provider lediglich einmal täglich abgefragt."
-                        ]
-                    },
-                    {
-                        "title": "Wenn ich die App außerhalb eines WLANs nutze, kommen dann Kosten für die Übertragung mobiler Daten auf mich zu?",
-                        "anchor": "mobile_data_costs",
-                        "active": false,
-                        "textblock": [
-                            "Die deutschen Netzbetreiber haben sich zu einem sog. Zero-Rating verpflichtet. Das bedeutet, dass für die Nutzung der Corona-Warn-App das mobile Datenvolumen sowohl in Deutschland als auch innerhalb des EU-Auslands kostenfrei zur Verfügung gestellt wird. Die mobilen Daten, welche
```
Comment (MikeMcC399) [inline src/data/faq_de.json]: "ihrerseits" => "Ihrerseits"

Comment (MikeMcC399) [inline src/data/faq_de.json]: "Covid-19" => "COVID-19"

Comment (Ein-Tim) [reply src/data/faq_de.json]: @MikeMcC399 It seems like you are reviewing deletions (GitHub has a red background for them) and all your reviews show "Outdated". No idea why, but just FYI.

Comment (Ein-Tim) [reply src/data/faq_de.json]: There must be something wrong here, this line is:

"Die deutschen Netzbetreiber haben sich zu einem sog. Zero-Rating verpflichtet. Das bedeutet, dass für die Nutzung der Corona-Warn-App das mobile Datenvolumen sowohl in Deutschland als auch innerhalb des EU-Auslands kostenfrei zur Verfügung gestellt wird. Die mobilen Daten, welche die Corona-Warn-App überträgt, um Diagnoseschlüssel ('Positivkennungen') hochzuladen oder im Fall einer Risiko-Begegnung vom Backend-Server benachrichtigt zu werden, kosten Sie keinen Cent und werden nicht von Ihrem vertraglichen Datenvolumen abgezogen."

There is no "Covid-19" in this line, or am I overlooking something?

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Since there was some activity on this PR today, perhaps you are considering using it soon?

I made some detailed comments for correction.

GitHub is showing major changes in the .json files which is a little worrying. If it stays this way, then merging the PR could breaks the FAQs. This should be checked out carefully.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

There is something technically wrong with this PR on GitHub. I'm not sure what is going wrong!

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Yes, this should be looked into before further changes are made!

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Please review what went wrong with this PR. It displays many changes on `faq.json` and `faq_de.json` and reviews are not shown correctly.
If this can't be solved in this PR, a new PR should be opened.

**Comment by MikeMcC399** [comment]
@brifemu 

The PR needs to be cleanly rebased on master. There are major merge conflicts.

**Comment by MikeMcC399** [comment]
I have added my failed comments from this PR to the new PR #2313.

This PR can be closed now.

**Pull request closed without merging**